src/main/java/com/inorg/services/product/controller/ShoppingController.java

```diff
@@ -0,0 +1,52 @@
+package com.inorg.services.product.controller;
+
+import com.commercetools.api.models.shopping_list.ShoppingList;
+import com.commercetools.api.models.shopping_list.ShoppingListPagedQueryResponse;
+import com.inorg.services.product.dto.LineItemDTO;
+import com.inorg.services.product.dto.ShoppingListDTO;
+import com.inorg.services.product.dto.UpdateItemQuantityDTO;
+import com.inorg.services.product.service.ShoppingService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@Validated
+@RequestMapping(path = "/shoppingLists", produces = APPLICATION_JSON_VALUE)
+public class ShoppingController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CustomerController.class);
+    private final ShoppingService shoppingService;
+
+    public ShoppingController(ShoppingService shoppingService) {
+        this.shoppingService = shoppingService;
+    }
+
+    @PostMapping(value = "/create", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ShoppingList createShoppingList(@RequestBody ShoppingListDTO shoppingListDTO) {
+        LOG.info("Creating a Shopping List");
+        return shoppingService.createShoppingList(shoppingListDTO);
+    }
+
+    @PostMapping(value = "/addLineItem", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Shopping List id should be path parameter and DTO should have only SKU and Qty. That is cleaner way to create REST end point
POST /shoppingLists/{shoppingListId}/lineItem